index.src.html

```diff
@@ -1182,11 +1238,11 @@ <h4 algorithm id="create-passwordcredential">
 
     2.  If |options|["{{CredentialCreationOptions/password}}"] is an {{HTMLFormElement}}, return the
         result of executing <a abstract-op>Create a `PasswordCredential` from an
-        `HTMLFormElement`</a> on |options|["{{CredentialCreationOptions/password}}"].
+        `HTMLFormElement`</a> given |options|["{{CredentialCreationOptions/password}}"] and |origin|.
```

Comment:
We can't access `HTMLFormElement`s from *in parallel*, so this needs to run from an algorithm that [[Create]] returns.

Reply:
not even if they are passed-in ?
so this is an overlooked prior bug in the spec ?

Reply:
Yeah, since most fields of the `HTMLFormElement` might be mutated by main-thread operations.

Reply:
I recommend addressing this in a subsequent PR after this PR is merged-to-master, as noted in the 20-Oct update here: https://github.com/w3c/webappsec-credential-management/pull/100#issue-137597236